### ISSUE TITLE
Crash Select in a Table with Graphic Field

### ISSUE THREAD

**Issue by JosepMariaGomez** [leonhad/paradoxdriver#19]
Hi again...
When I try to execute a simple query over a table with a Graphic field (or more than one field) the following exception is raised:

```
java.sql.SQLException: Field type unsupported.
	at com.googlecode.paradox.data.FieldFactory.parse(FieldFactory.java:87)
	at com.googlecode.paradox.data.TableData.readRow(TableData.java:294)
	at com.googlecode.paradox.data.TableData.loadData(TableData.java:131)
	at com.googlecode.paradox.planner.plan.SelectPlan.loadTableData(SelectPlan.java:332)
	at com.googlecode.paradox.planner.plan.SelectPlan.execute(SelectPlan.java:125)
	at com.googlecode.paradox.ParadoxStatement.executeSelect(ParadoxStatement.java:478)
	at com.googlecode.paradox.ParadoxStatement.executeQuery(ParadoxStatement.java:216)
	at com.googlecode.paradox.ParadoxStatement.executeQuery(ParadoxStatement.java:216)
```
I've been searching and I found that Phoenix arranged a similar issue long time ago (2017), but this exception is different...

the code used is quite simple..

```
                ResultSet rs = st.executeQuery(""
                        + "SELECT * \n"
                        + "  FROM myTable \n"
                        + "");
```

TIA

**Comment by przemek-mela** [comment]
I have the same problem.
`Class.forName("com.googlecode.paradox.Driver");
        java.sql.Connection conn = DriverManager.getConnection("jdbc:paradox:./db");
        Statement stmt=conn.createStatement();  
        ResultSet rs=stmt.executeQuery("select * from users");  
        while(rs.next())  
        System.out.println(rs.getInt(1)+"  "+rs.getString(2)+"  "+rs.getString(3));  
        conn.close();  `

this code results with an error:
`Exception in thread "main" java.sql.SQLException: Field type unsupported.
	at com.googlecode.paradox.data.FieldFactory.parse(FieldFactory.java:87)
	at com.googlecode.paradox.data.TableData.readRow(TableData.java:294)
	at com.googlecode.paradox.data.TableData.loadData(TableData.java:131)
	at com.googlecode.paradox.planner.plan.SelectPlan.loadTableData(SelectPlan.java:332)
	at com.googlecode.paradox.planner.plan.SelectPlan.execute(SelectPlan.java:125)
	at com.googlecode.paradox.ParadoxStatement.executeSelect(ParadoxStatement.java:478)
	at com.googlecode.paradox.ParadoxStatement.executeQuery(ParadoxStatement.java:216)`

Please help.

**Comment by leonhad** [comment]
Fixed in version 1.5.0.
